server/index.d.ts

```diff
@@ -2,16 +2,16 @@ import BaseServer, { Logger, Reporter, BaseServerOptions } from '../base-server'
 
 export type ServerOptions = BaseServerOptions & {
   /**
-   * Report process/errors to CLI in bunyan JSON or in human readable
+   * Report process/errors to CLI in pino JSON or in human readable
    * format. It can be also a function to show current server status.
    * Default is `'human'`.
    */
   reporter?: 'human' | 'json' | Reporter
 
   /**
-   * Bunyan logger with custom settings.
+   * Logger with custom settings.
```

Comment:
Can I ask you to add a little example here?

We didn’t have it before, but it will be nice to show how to set own logger. As a pino’s user you will do it better.

You can use some real pino logger (for instance, for Google Cloud).

Reply:
I will try to add an example

Reply:
Maybe I didn't get you point about example. To discuss it I added simple example with elastic search. But in essence any logger that conforms to specified type annotation, not only pino one

Reply:
It looks good. You just need to fix code style in example to follow Logux rules and add somthing link:

```js
const server = new Server(
  Server.loadOptions(process, {
    …,
    logger: pino({ level: 'info' }, streamToElastic)
  })
)
```

Reply:
Not so sure about whether I spot all typos